src/app/projects/page.js

```diff
@@ -114,14 +114,29 @@ export default async function Projects() {
     orgUrl: responseData.viewer.url,
     ...project
   }))
-  const orgProjects = responseData.viewer.organizations.nodes.map(org =>
-    org.projectsV2.nodes.map(project => ({
-      ownerType: 'org',
-      orgName: org.login,
-      orgUrl: org.url,
-      ...project
-    }))
-  ).flat()
+
+  const orgProjects = responseData.viewer.organizations.nodes.reduce((projectsResult, org) => {
```

Comment:
Maybe, in the future, we want to show a tooltip to the user mentioning it has not enough permissions so no projects are shown for that org.

Reply:
@smoya what was the project that you didn't had permission to see, is it because you didn't granted the permission during auth to the org ?

Reply:
I did not grant permission (or ask permission to admins) on some orgs I belong to, so projects were not fetched from GH API (came as literal `null` value)